HISTORY.md

```diff
@@ -52,6 +54,8 @@
   * Change neural network types to avoid unnecessary use of rvalue references
     (#2259).
 
+  * Add L1 Loss function (#2203)
```

Comment:
Hey @himanshupathak21061998, I think we only need to add once. Could remove either line 38 or 57 please. Thanks a lot for this great PR.

Reply:
Ahh! again issue because of rebasing thanks for noticing @kartikdutt18

Reply:
No worries. Thanks a lot for this awesome PR. Could you resolve these conversations, I am not able to do it from my side.